### PR TITLE
First nastpec comment

### PR DESCRIPTION
**Pull request by JuliusBrain**
A short summary.

Important changes:
- [x] Something worth noting for reviewers.

Mandatory:
- [x] Added documentation.

Closes #

### CODE REVIEW

**Comment by JuliusBrain** [comment]
@rgeraldes Could you take a look at those comments and write explanations for some of the params for ValdiatorMgr.sol? I don't know I got them right.

**Comment by JuliusBrain** [comment]
@rgeraldes Could you take a look at those comments and write explanations for some of the params for ValdiatorMgr.sol? I don't know I got them right.

File: client/contracts/truffle/contracts/oracle/OracleMgr.sol [hunk 1]
```diff
@@ -149,28 +184,43 @@ contract OracleMgr is Pausable {
...
     }
     
+    /**
+     * @dev Get deposit count
+     */
     function getDepositCount() public view returns (uint count) {
         return oracleRegistry[msg.sender].deposits.length; 
     }
-
+    /**
```
Comment (JekaMas) [inline client/contracts/truffle/contracts/oracle/OracleMgr.sol]: Shall we add a blank line between `}` and comments?

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 2]
```diff
@@ -59,7 +61,14 @@ contract ValidatorMgr is Pausable {
         require(!isValidator(tkn.sender));
         _;
     }
-
+    /**
+     * Constructor.
+     * @param _baseDeposit base deposit for Oracle.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: *Validator

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 3]
```diff
@@ -59,7 +61,14 @@ contract ValidatorMgr is Pausable {
...
+     * @param _maxNumValidators Maximum numbers of Validators.
+     * @param _freezePeriod Freeze period for Validator.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: Validator's deposits

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 4]
```diff
@@ -59,7 +61,14 @@ contract ValidatorMgr is Pausable {
...
+     * @param _miningTokenAddr Address of mining token.
+     * @param _superNodeAmount Amount of super nodes.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: Amount required to be considered a super node

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 5]
```diff
@@ -70,25 +79,44 @@ contract ValidatorMgr is Pausable {
...
+     */
     function isSuperNode(address code) public view returns (bool isIndeed) {
         if (!isValidator(code)) return false;
 
         Deposit[] deposits = validatorRegistry[code].deposits;
         return deposits[deposits.length - 1].amount >= superNodeAmount;
     }
 
+    /**
+     * @dev gets validator count
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: consistency: gets vs Get (below)

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 6]
```diff
@@ -99,8 +127,10 @@ contract ValidatorMgr is Pausable {
         return (maxNumValidators - validatorPool.length) > 0;
     }
 
-    // getMinimumDeposit returns the base deposit if there are positions available or
-    // the current smallest deposit required if there aren't positions availabe.
+    /**
+     * @dev returns the base deposit if there are positions available or
+            the current smallest deposit required if there aren't positions availabe.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: available

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 7]
```diff
@@ -111,10 +141,18 @@ contract ValidatorMgr is Pausable {
...
+     * @dev updates the checksum
+     */
     function _updateChecksum() private {
         validatorsChecksum = keccak256(validatorPool);
     }
 
+    /**
+     * @dev Add new validator
+     * @param code
+     * @param deposit deposit amount
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: deposit deposit

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 8]
```diff
@@ -138,10 +176,17 @@ contract ValidatorMgr is Pausable {
         _updateChecksum();
     }
 
+    /**
+     * @dev Sets new base deposit for validators
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: consistency: Set vs Sets

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 9]
```diff
@@ -165,31 +213,51 @@ contract ValidatorMgr is Pausable {
...
+     */
     function _registerValidator() private whenNotPaused onlyNewCandidate onlyWithMinDeposit {
         if (!_hasAvailability()) {
             _deleteSmallestBidder();
         }
         _insertValidator(tkn.sender, tkn.value);
     }
 
+    /**
+     * @dev deregister validator
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: consistency: caps vs no caps

File: client/contracts/truffle/contracts/oracle/OracleMgr.sol [hunk 10]
```diff
@@ -55,7 +57,15 @@ contract OracleMgr is Pausable {
         require(_price > 0);
         _;
     }
-
+    /**
+     * Constructor.
+     * @param _initialPrice Initial Price.
+     * @param _baseDeposit base deposit for Oracle.
+     * @param _maxNumOracles Maximum numbers of Oracles.
+     * @param _freezePeriod Freeze period for Oracle.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/oracle/OracleMgr.sol]: Oracle's deposits

File: client/contracts/truffle/contracts/oracle/OracleMgr.sol [hunk 11]
```diff
@@ -127,18 +153,27 @@ contract OracleMgr is Pausable {
         }
     }
 
+    /**
+     * @dev Get Oracle count
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/oracle/OracleMgr.sol]: consistency: caps or no caps (Oracle) - vs below

File: client/contracts/truffle/contracts/oracle/OracleMgr.sol (hunk 10 above)
Comment (JuliusBrain) [reply client/contracts/truffle/contracts/oracle/OracleMgr.sol]: FreezePeriod is "Oracle's deposits"? Or you meant something different?

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 12]
```diff
@@ -60,6 +63,14 @@ contract ValidatorMgr is Pausable {
         _;
     }
 
+    /**
+     * Constructor.
+     * @param _baseDeposit Validator's deposits.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: base deposit is different than validator's deposits

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 13]
```diff
@@ -60,6 +63,14 @@ contract ValidatorMgr is Pausable {
...
+     * @param _maxNumValidators Maximum numbers of Validators.
+     * @param _freezePeriod Freeze period for Validator.
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: freeze period for validator's deposit

File: client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol [hunk 14]
```diff
@@ -138,10 +178,17 @@ contract ValidatorMgr is Pausable {
...
 
+    /**
+     * @dev Set new base deposit for Validators
+     */
     function setBaseDeposit(uint deposit) public onlyOwner {
         baseDeposit = deposit;
     }
 
+    /**
+     * @dev Set maximum of Validators
```
Comment (rgeraldes) [inline client/contracts/truffle/contracts/consensus/mgr/ValidatorMgr.sol]: Set maximum number of validators

**Pull request merged**